apps/E2E/package.json

```diff
@@ -39,14 +39,14 @@
     "@rnx-kit/metro-config": "^1.3.1",
     "@types/jasmine": "5.1.4",
     "@types/react": "^18.2.0",
-    "@wdio/appium-service": "8.29.1",
-    "@wdio/cli": "8.35.1",
-    "@wdio/globals": "8.35.1",
-    "@wdio/jasmine-framework": "8.35.1",
-    "@wdio/json-reporter": "8.32.4",
-    "@wdio/local-runner": "8.35.1",
-    "@wdio/spec-reporter": "8.32.4",
-    "appium": "^2.5.1",
+    "@wdio/appium-service": "^8.40.0",
+    "@wdio/cli": "^8.40.0",
+    "@wdio/globals": "^8.40.0",
+    "@wdio/jasmine-framework": "^8.40.0",
+    "@wdio/json-reporter": "^8.40.0",
+    "@wdio/local-runner": "^8.40.0",
+    "@wdio/spec-reporter": "^8.40.0",
+    "appium": "^2.11.2",
```

Comment:
Side note: I wonder if we can publish a dependency profile like we do with the rest of FURN for the stable sets of Webdriver deps?

Reply:
That's a good callout - right now, v8 is stable (they're already releasing v9 soon). Once v9 comes out, I'm sure we can do this.